inst/include/common/model.hpp

```diff
@@ -67,18 +67,28 @@ namespace fims_model {
             // vector< vector<Type> > creates a nested vector structure where
             // each vector can be a different dimension. Does not work with ADREPORT
             // fleets
-            vector<vector<Type> > exp_index(n_fleets);
-            vector<vector<Type> > exp_catch(n_fleets);
-            vector<vector<Type> > cnaa(n_fleets);
-            vector<vector<Type> > cwaa(n_fleets);
-            vector<vector<Type> > cnal(n_fleets);
-            vector<vector<Type> > pcnaa(n_fleets);
-            vector<vector<Type> > pcnal(n_fleets);
```

Comment:
if the difference between proportion and composition is whether it is data or the predicted value, should probably be consistent with index/exp_index and landings/exp_landings.

Reply:
sorry should be index_exp and landings_exp, I think

Reply:
This one is confusing because the comp data can either be in real numbers or proportions. Internally the proportion is what is used for the expected likelihood calculation but on the user end this is hard to compare because we don't report the observed values as proportion. We could refactor to report observed and expected in both units I guess. I did try to refactor code throughout to have expected values be clearer but there is still more to do.